src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int BALLS_QUANTITY = 101;
```

Comment:
101 is not a balls quantity
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/17576664-6560-44ee-b884-ec091abc6966)